jwst/datamodels/schemas/msa_targ_acq.schema.yaml

```diff
@@ -0,0 +1,200 @@
+        unit: counts
+      - name: box_peak_col
+        datatype: int16
+        unit: pixels
+      - name: box_peak_row
+        datatype: int16
+        unit: pixels
+      - name: box_total_value
+        datatype: float32
+        unit: counts
+      - name: iterations
+        datatype: int16
+      - name: final_iter_col_diff
+        datatype: float32
+        unit: pixels
+      - name: final_iter_row_diff
+        datatype: float32
+        unit: pixels
+      - name: corrected_col
+        datatype: int16
+        unit: pixels
+      - name: corrected_row
+        datatype: int16
+        unit: pixels
+      - name: image_centroid_col
+        datatype: float32
+        unit: pixels
+      - name: image_centroid_row
+        datatype: float32
+        unit: pixels
+      - name: detector_centroid_col
+        datatype: float32
+        unit: pixels
+      - name: detector_centroid_row
+        datatype: float32
+        unit: pixels
+      - name: sci_x_centroid
+        datatype: float32
+        unit: pixels
+      - name: sci_y_centroid
+        datatype: float32
+        unit: pixels
+      - name: col_second_moment
+        datatype: float32
+        unit: pixels^2
+      - name: row_second_moment
+        datatype: float32
+        unit: pixels^2
+      - name: col_third_moment
+        datatype: float32
+        unit: pixels^3
+      - name: row_third_moment
+        datatype: float32
+        unit: pixels^3
+      - name: convergence_status
+        datatype: [ascii, 20]
+      - name: found_status
+        datatype: [ascii, 10]
+      - name: found_v2
+        datatype: float32
+        unit: arcseconds
+      - name: found_v3
+        datatype: float32
+        unit: arcseconds
+      - name: found_msa_x
+        datatype: float32
+        unit: arcseconds
+      - name: found_msa_y
+        datatype: float32
+        unit: arcseconds
+      - name: lsf_removed_status
+        datatype: [ascii, 20]
+      - name: lsf_removed_reason
+        datatype: [ascii, 14]
+      - name: lsf_removed_x
+        datatype: float32
+        unit: arcseconds
+      - name: lsf_removed_y
+        datatype: float32
+        unit: arcseconds
+      - name: fit_offset_v2
+        datatype: float32
+        unit: arcseconds
+      - name: fit_offset_v3
+        datatype: float32
+        unit: arcseconds
+      - name: x_rel
+        datatype: float32
+        unit: arcseconds
+      - name: y_rel
+        datatype: float32
+        unit: arcseconds
+      - name: xrot_rel
+        datatype: float32
+        unit: arcseconds
+      - name: yrot_rel
+        datatype: float32
+        unit: arcseconds
+      - name: v2_new_from_fit
+        datatype: float32
+        unit: arcseconds
+      - name: v3_new_from_fit
+        datatype: float32
+        unit: arcseconds
+      - name: v2_fit_residual
+        datatype: float32
+        unit: arcseconds
+      - name: v3_fit_residual
+        datatype: float32
+        unit: arcseconds
+    meta:
```

Comment:
I presume this meta section is for defining the extension header keywords. There are quite a few not included here.

Reply:
What other keywords beside those from :

- $ref: core.schema

 are needed ?

Reply:
@jbdoggett where can we get a complete list of the non-table related keywords that belong in the header of this extension? We don't need any of the table column definition keywords in our schema (e.g. TFORM, TTYPE, TUNIT, etc.). Those are handled automatically by the datamodels table interface, based on the column definitions in the top part of the schema shown here.

Reply:
They're in the technical memo (attached to JSDP-1751 referenced in JP-2369) and the keyword dictionary JSON file. While the technical memo may not be accurate in definitions, I'm pretty sure it's complete and is easy to read. The JSON file is complete and accurate.

Reply:
Ah yes, I see what you mean. There are many entries in https://jira.stsci.edu/secure/attachment/71940/after-core.targacq.msa.schema.json, like TASTATUS, STAT_RSN, CRMETHOD, etc., that aren't yet in our schema.

Reply:
I will add those needed keywords also.